Ship_Game/GameScreens/ShipDesign/ShipDesignScreen.cs

```diff
@@ -640,5 +644,16 @@ void RefreshHullSelectList()
                 }
             }
         }
+
+        protected override void ResetMusic()

```

Comment:
```cs
void StartMusic()
{
    ScreenManager.StopMusic();
    ScreenManager.PlayMusic("ShipyardTheme");
}
```
I think `GlobalStats.Defaults.CustomMenuMusic` is not relevant here and actually we should remove it from the game since we now have AudioConfig

Reply:
done